src/icons/flowTypes.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+export type Icon = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+export type TwoTonedIcon = {
+    className?: string,
+    height?: number,
+    styleProps: {|
+        backgroundColor?: string,
+        foregroundColor?: string,
+    |},
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
```

Comment:
Should this be a junction type of a base type containing className, height, title, and width?

Reply:
Updated this type now. The `TwoTonedIcon` doesn't include `color` since we decided that will be overridden with CSS only. I think it's best to keep them separate because of that.